CONTRIBUTING.adoc

```diff
@@ -0,0 +1,32 @@
+= Contribution Guide
+
+🎉 Thanks for taking the time to contribute! 🎉 Contributions are welcome from
+anyone on the internet, and even the smallest of fixes are appreciated!
+
+The following is a set of guidelines for contributing. These are mostly
+guidelines, not rules. Use your best judgment, and feel free to propose changes
+to this document in a pull request.
+
+== Getting started
+
+1. Fork https://github.com/keep-network/keep-common[`keep-network/keep-common`]
+2. Clone your fork
+3. Follow the
+https://github.com/keep-network/keep-common/blob/main/README.adoc[installation
+steps] in the README.
+4. Enable
+https://help.github.com/en/articles/about-commit-signature-verification[commit
+signing].
+4. Setup the <<Development Tooling>>.
```

Comment:
Should the Development Tooling link somewhere?

Reply:
ah, good catch; that was a holdover from when there was a Development Tooling section, and then I cut it

Reply:
addressed in [9ae1a2](https://github.com/keep-network/keep-common/pull/114/commits/9ae1a29ccb8da610b01ef9f7d1f53e1475bf0b38)